### ISSUE TITLE
the highlighted item won't follow your cursor

### ISSUE THREAD

**Issue by tcstory** [zurb/tribute#221]
**How can we reproduce this bug?**

mac os, chrome 73

when you hover over the menu items, the highlighted item likely won't change,

**Link (jsfiddle/plunkr/codepen) or Screenshot:**

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/5856466/54821059-1cba5380-4cdc-11e9-87a1-23da6c4eccf7.gif)

as you can see, when i  hover over the items, the highlighted item likely won't change,

i looked at the source code, and i found that the line 267 of Tribute.js had the following code:

```js
li.addEventListener('mouseenter', (e) => {
  let li = e.target;
  let index = li.getAttribute('data-index')
  if (e.movementX !== 0 && e.movementY !== 0) {
    this.events.setActiveLi(index)
  }
})
```

what is the use of the code `e.movementX !== 0 && e.movementY !== 0`, i sure that this code cause this problem

**Comment by MalayaliRobz** [comment]
Could you share a jsfiddle/plunkr/codepen link replicating the issue? Highlighting seems to be working fine on the demo page(https://zurb.github.io/tribute/example/).

`e.movementX !== 0 && e.movementY !== 0` is for
fixing bug that caused setActiveLi to be called when mouse pointer inside one of the items and scrolling through keyboard, caused improper highlighting.
![52480095-c03e2180-2bd0-11e9-9c54-e67a6f465728](https://user-images.githubusercontent.com/21201812/54810383-ff6d9100-4cab-11e9-8d71-c25efb597609.gif)

**Comment by tcstory** [comment]
@MalayaliRobz sorry, i had post the gif just now.

you shouldn't use keyboard to select item, instead of using cursor

**Comment by MalayaliRobz** [comment]
Got it. Seems that mouseenter.X and Y are being zero.

**Comment by MalayaliRobz** [comment]
have a fix here https://github.com/zurb/tribute/pull/222

**Comment by mrsweaters** [comment]
Fixed, releasing in 3.7.0
